README.md

```diff
@@ -0,0 +1,38 @@
+# ServiceAngularTemplate
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.20.
+
+# %CUSTOM_PLUGIN_SERVICE_NAME%
+This project is a custom angular 2+ service created by the Mia-Platform API Console.
```

Comment:
I'd not say it's "angular 2+"; it's "Angular". 

We're actually using Angular 8 so either we write Angular or Angular 8